internal/providers/terraform/parser.go

```diff
@@ -150,9 +158,12 @@ func (p *Parser) parseJSON(j []byte, usage map[string]*schema.UsageData) ([]*sch
 	conf := parsed.Get("configuration.root_module")
 	vars := parsed.Get("variables")
 
-	pastResources := p.parseJSONResources(true, baseResources, usage, parsed, providerConf, conf, vars)
 	resources := p.parseJSONResources(false, baseResources, usage, parsed, providerConf, conf, vars)
+	if p.loadPriorState {
```

Comment:
Is the purpose of the `loadPriorState option` to avoid loading `pastResources` if we don't need to, or is there more to it?